airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -397,15 +396,27 @@ def __init__(
         super().__init__(**kwargs)
         self.cluster_identifier = cluster_identifier
         self.aws_conn_id = aws_conn_id
+        # These parameters are added to address an issue with the boto3 API where the API
+        # prematurely reports the cluster as available to receive requests. This causes the cluster
+        # to reject initial attempts to resume the cluster despite reporting the correct state.
+        self.attempts = 10
+        self.attempt_interval = 15
```

Comment:
should we mark as private?
since these are not user facing I'd rather users not to relay on this (In case they subclass the operator)

Reply:
good point